### ISSUE TITLE
I used waybar to switch topics and then waybar crashed

### ISSUE THREAD

**Issue by 220320bs** [prasanthrangan/hyprdots#18]
This is the terminal under the enable waybar

![swappy-20230623_085347](https://github.com/prasanthrangan/hyprdots/assets/113283361/50b808ec-3ecd-48ef-aa16-12e027926499)

**Comment by prasanthrangan** [comment]
Hi, was there any change done to `~/.config/waybar/config.jsonc` or `~/.config/waybar/style.css` config files?
Also regarding the `No icon name or pixmap given` error, I'm unable to replicate it, so please share any steps you followed -

![image](https://github.com/prasanthrangan/hyprdots/assets/106020512/ed1f87a9-9285-4a6e-ad99-a7eb2d6cb1b2)

so can you try disabling the icon in config.jsonc file as below -

```
    "hyprland/window": {
        "format": "  {}",
        "separate-outputs": true,
        "icon": false // try disabling icons by adding this line
        "rewrite": {
            "tittu@kryptonite:(.*)": "$1 ",
            "(.*) — Mozilla Firefox": "$1 󰈹",
            "(.*)Mozilla Firefox": "Firefox 󰈹",
            "(.*) - Visual Studio Code": "$1 󰨞",
            "(.*)Visual Studio Code": "Code 󰨞",
            "(.*) — Dolphin": "$1 󰉋",
            "(.*)Spotify": "Spotify 󰓇",
            "(.*)Steam": "Steam 󰓓"
          }
```

**Comment by 220320bs** [comment]
I didn't change the configuration, I tried disabling the icon but still the same, I think it should be a problem with my computer or waybar

**Comment by prasanthrangan** [comment]
The error is due to `killall -SIGUSR2 waybar` as its unable to reload waybar style... Fixed this in commit c4019f7
